node/runtime/src/lib.rs

```diff
@@ -136,6 +141,10 @@ parameter_types! {
 impl babe::Trait for Runtime {
 	type EpochDuration = EpochDuration;
 	type ExpectedBlockTime = ExpectedBlockTime;
+	type IdentificationTuple = historical::IdentificationTuple<Self>;
```

Comment:
this is redundant because it can be accessed by the `KeyOwnerSystem`. (it should be called `KeyOwnerProofSystem`)